src/simsopt/field/tracing.py

```diff
@@ -398,7 +398,7 @@ class IterationStoppingCriterion(sopp.IterationStoppingCriterion):
     pass
 
 
-def plot_poincare_data(fieldlines_phi_hits, phis, filename, mark_lost=False, aspect='auto', dpi=300):
+def plot_poincare_data(fieldlines_phi_hits, phis, filename, mark_lost=False, aspect='equal', dpi=300):
```

Comment:
why the change?

Reply:
Having aspect='auto' tends to distort flux surfaces. For example, if the aspect is not 'equal', a circular flux surface will appear elliptical.